native/packages/react-native-bpk-component-chip/src/BpkChip.js

```diff
@@ -0,0 +1,55 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import BpkChipWrapper from './BpkChipWrapper';
+import {
+  type Props as CommonProps,
+  commonPropTypes,
+  commonDefaultProps,
+} from './common-types';
+
+type Props = {
+  ...$Exact<CommonProps>,
+  large: boolean,
+  selected: boolean,
+};
+
+const BpkChip = (props: Props) => {
+  const { ...rest } = props;
+
+  return <BpkChipWrapper {...rest} dismissible={false} />;
```

Comment:
This could just be a function call instead of a render which reduces the virtual DOM tree by one level

```javascript
  return BpkChipWrapper({ dismissible: false, ...rest });
```

Reply:
Note that prop types are ignored in this case... not sure if it's worth it.

Reply:
Only the `BpkChipWrapper` prop types are ignored right? They are internal anyway?

Reply:
Strictly speaking we don't need `propTypes` for `BpkChipWrapper` since we have Flow and it's internal?

Reply:
Regardless if it's internal or not, the same props are declared for this component anyways so ...

I still think this stands though:
> Also I think it's kind of surprising for devs who haven't came across squashing before.

Reply:
I do like the idea of a smaller DOM tree but I wouldn't want to prematurely optimise at the expense of principle of least surprise (hey I'm a poet)

Reply:
Okay let's not do that then :)